doc/changes/latest.inc

```diff
@@ -36,6 +36,7 @@ Enhancements
 
 Bugs
 ~~~~
+- Fix bug in :func:`mne.io._edf._export_raw` when exporting raw to EDF with EDFLib when subject `sex` is missing (:gh:`11375` by :newcontrib:`Mark Alexander Henney`)
```

Comment:
We try to document using the public name of a function because 1) this is where end-users will see the effects, and 2) it links properly. Then double-backticks for code mode for RST (as opposed to single-backticks for MD) 
```suggestion
- Fix bug in :func:`mne.export.export_raw` when exporting raw to EDF with EDFLib when subject ``sex`` is missing (:gh:`11375` by :newcontrib:`Mark Alexander Henney`)
```